tests/parallel/mapping-nearest-neighbor-gradient/GradientTestParallelScalar.cpp

```diff
@@ -0,0 +1,89 @@
+#ifndef PRECICE_NO_MPI
+
+#include <Eigen/Core>
+#include <algorithm>
+#include <deque>
+#include <fstream>
+#include <istream>
+#include <iterator>
+#include <memory>
+#include <ostream>
+#include <string>
+#include <vector>
+
+#include "action/RecorderAction.hpp"
+#include "logging/LogMacros.hpp"
+#include "math/constants.hpp"
+#include "math/geometry.hpp"
+#include "mesh/Data.hpp"
+#include "mesh/Mesh.hpp"
+#include "mesh/SharedPointer.hpp"
+#include "mesh/Utils.hpp"
+#include "mesh/Vertex.hpp"
+#include "precice/SolverInterface.hpp"
+#include "precice/impl/MeshContext.hpp"
+#include "precice/impl/Participant.hpp"
+#include "precice/impl/SharedPointer.hpp"
+#include "precice/impl/SolverInterfaceImpl.hpp"
+#include "precice/types.hpp"
+#include "testing/TestContext.hpp"
+#include "testing/Testing.hpp"
+
+using namespace precice;
+using precice::testing::TestContext;
+
+BOOST_AUTO_TEST_SUITE(PreciceTests)
```

Comment:
The main test suite was renamed to `Integration` in https://github.com/precice/precice/commit/8b51a94a2ca69cbbd0ff0b4e2d4c165769aefe9f.